src/Symfony/Component/Security/Core/Security.php

```diff
@@ -22,4 +26,50 @@
     const AUTHENTICATION_ERROR = '_security.last_error';
     const LAST_USERNAME = '_security.last_username';
     const MAX_USERNAME_LENGTH = 4096;
+
+    private $container;
+
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
+    }
+
+    /**
+     * @return UserInterface|null
+     */
+    public function getUser()
```

Comment:
Return type instead of phpdoc?

Reply:
No, this targets 3.4 which supports PHP 5.6.